source/FpsConfig.h

```diff
@@ -71,7 +71,12 @@ class PlayerConfig {
 	Vector2			turnScale = Vector2(1.0f, 1.0f);			///< Player (horizontal/vertical) turn rate scaler
 	Array<bool>		axisLock = { false, false, false };			///< World-space player motion axis lock
 	bool			stillBetweenTrials = false;					///< Disable player motion between trials?
-	bool			resetPositionPerTrial = false;				///< Reset the player's position on a per trial basis (to scene default)
+	bool			resetPositionPerTrial = false;				///< Reset the player's position on a per trial basis (to scene default)'
+	float			sprintMultiplier = 2.0f;					///< Multiplies with players move rate when player is sprinting (defaults to 2x)
+	float			headBobAmplitude = 0.1f;					///< Determines how high/low players head will oscillate during movement (defaults to 0.1)
+	float			headBobFrequency = 1.0f;					///< Determines how fast players head will oscillate during movement (defaults to 1)
+	float			movementAcceleration = 0.05f;				///< Player acceleration during movement (defaults to 0.05)
+	float			movementDeceleration = 0.15;				///< Player deceleration while stopping (defaults to 0.15)
```

Comment:
One more time same as above, I'd start w/ this disabled. Also make line 79 `0.15f`.

Reply:
Updated.